### DISCUSSION TITLE
New dashboard not mobile friendly.

### DISCUSSION THREAD

**Discussion by warrmr** [apache/skywalking#8871] [General]
We are in the process of trialing Skywalking in our apps and we love it so far the response time from the developers on issues is amazing and the charts and ease of first time setup is awesome. We got a few issues to iron out but the overall experience is great. 

So much so I cant stop checking the dashboard looking at all the pretty graphs :) However the dashboard ui isn't very mobile friendly (see screenshot) is it in the pipeline to make it nicer for mobile please ;) 

![Image from iOS (1)](https://user-images.githubusercontent.com/393693/163282747-90ec2c70-b0c1-4cf7-b88b-8bfb34516833.jpg)

**Comment by wu-sheng** [comment]
For now, it isn't. Yes. Because v8 isn't too. We may deal with it later.

If anyone wants to help, please let us know.

**Comment by wu-sheng** [comment]
@Fine0830 I think we only have timr to consider the mobile layout after ebpf profiling panded.

**Reply by Fine0830** [reply]
Agree. I will make the new ui adapt mobile terminal.

**Comment by wu-sheng** [comment]
@warrmr Check this https://github.com/apache/skywalking-booster-ui/pull/78

**Reply by warrmr** [reply]
That looks awesome, cant wait for 9.1.0 to come out so i can test it :)
